pkg/minikube/problem/err_map.go

```diff
@@ -119,6 +125,12 @@ var vmProblems = map[string]match{
 		URL:    "http://mikko.repolainen.fi/documents/virtualization-with-kvm",
 		Issues: []int{2991},
 	},
+	"OLD_KVM_VERSION": {
+		Regexp: re(`Too old KVM version found`),
+		Advice: "too old KVM version. Jump forward to new valid KVM version as a workaround.",
+		URL:	"https://github.com/kubernetes/minikube/blob/master/docs/drivers.md"
```

Comment:
```suggestion
		URL:	"https://github.com/kubernetes/minikube/blob/master/docs/drivers.md#kvm2-upgrade",
```

There's a specific link for kvm upgrade. Also missing a comma at the end.